### ISSUE TITLE
Previous/ Next buttons tooltip is not displayed with keyboard navigation

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4594]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.4.13 Content on Hover or Focus

**Test Environment:**
- Test URL: https://clarity.design/documentation/alerts
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to previous or next button provided for Multiple app level alerts using tab key navigation and check if tooltip is visible.

**Expected result:**
Tooltip should be displayed when user moves focus on to the previous or next button.

**Actual result:**
Tooptip is not displayed when previous or next buttons are focused with keyboard navigation. Tooltip is only visible with mouse hover. That means button labels for keyboard users only relies on shape of the button due to unavailability of tooltip.

**Proposed solution:**
Add keyboard event handlers along with mouse hover event handlers added for the buttons which triggers the tooltip.

**Reporter Email: ** gkhadke@vmware.com

**Comment by gkhadK** [comment]
Same issue is observed for Icon button group. Tooltip is not available for keyboard users. In addition to that, Users using mouse/trackpad had to precisely place the pointer on the icon see the tooltip.

**Comment by gkhadK** [comment]
Similar issue is observed for custom trigger clarity icon provided for signpost.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
